stock_picking_button_validate_hook/model/stock_picking.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 ForgeFlow, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, models, _
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    def _button_validate_get_no_quantities_done(self):
+        precision_digits = self.env['decimal.precision'].precision_get(
+            'Product Unit of Measure')
+        return all(
+            float_is_zero(
+                move_line.qty_done,
+                precision_digits=precision_digits) for move_line in
+            self.move_line_ids.filtered(
+                lambda m: m.state not in ('done', 'cancel')))
+
+    @api.model_cr
+    def _register_hook(self):
+
+        def new_button_validate(self):
+            self.ensure_one()
+            if not self.move_lines and not self.move_line_ids:
+                raise UserError(_('Please add some items to move.'))
+            # If no lots when needed, raise error
+            picking_type = self.picking_type_id
+            # START OF PATCH #
+            no_quantities_done = self._button_validate_get_no_quantities_done()
+            # no_quantities_done = all(
```

Comment:
Remove this commented code, already in the function `_button_validate_get_no_quantities_done`

Reply:
@AdriaGForgeFlow Thanks. Comment attended.